rest/data/interface.go

```diff
@@ -103,3 +115,12 @@ type BuildloggerOptions struct {
 	Tail          int
 	SoftSizeLimit int
 }
+
+// TestResultsOptions holds all values required to find a specific TestResults
```

Comment:
Can we name this something else since it looks so similar to `TestResultsFindOptions`?